apps/ledger-live-mobile/src/newArch/features/LandingPages/screens/LargeMoverLandingPage/index.tsx

```diff
@@ -49,6 +50,7 @@ export const LargeMoverLandingPage = ({ route }: LargeMoverLandingPageProps) =>
 
   return (
     <SafeAreaView style={{ flex: 1, backgroundColor: colors.neutral.c00 }}>
+      <TrackScreen name="Large Mover Landing Page" />
```

Comment:
Can we add the `currencyIds` and `initialRange` route parameters to this event please ? It will allow us to do a deeper analysis of the feature usage